src/qt/bitcoin.cpp

```diff
@@ -291,22 +301,14 @@ bool BitcoinApplication::baseInitialize()
 
 void BitcoinApplication::startThread()
 {
-    if(coreThread)
-        return;
-    coreThread = new QThread(this);
```

Comment:
In commit "qt: Add BitcoinCore::m_thread member" (6b769d9e9e1c73cef37826e3c842fe9b65737456)

The coreThread member doesn't actually do anything after this change. Would be less confusing to squash the next commit into this one and delete coreThread entirely instead of keeping it half-used.

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/381#issuecomment-880033334).